hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/contract/ContractCallTransitionLogic.java

```diff
@@ -255,6 +251,25 @@ private ResponseCodeEnum validateSemantics(final TransactionBody transactionBody
         return OK;
     }
 
+    // Determine if the operation should be sanity checked.
+    private boolean possiblySanityCheckOp(final ContractCallTransactionBody op) {
+        final var target = targetOf(op);
+
+        // Tokens are valid targets
+        if (entityAccess.isTokenAccount(target.toId().asEvmAddress())) {
+            return false;
+        }
+
+        // Check for possible lazy create
+        if (target.equals(EntityNum.MISSING_NUM)
+                && isOfEvmAddressSize(op.getContractID().getEvmAddress())
+                && op.getAmount() > 0) {
```

Comment:
Should this be ?  
```suggestion
              (target.equals(EntityNum.MISSING_NUM)
                && isOfEvmAddressSize(op.getContractID().getEvmAddress())
                ||op.getAmount() > 0
```

Reply:
I suppose it depends on how broad or narrow this precheck should be.  I don't see any not to narrow the scope as you suggest.  Will change as suggested.

Reply:
My comment was just based on the check that existed in mono-service. Not sure if it needs to be the same also.